addons/hw_drivers/tools/helpers.py

```diff
@@ -61,14 +61,33 @@ def run(self):
 elif platform.system() == 'Linux':
     @contextlib.contextmanager
     def writable():
-        subprocess.call(["sudo", "mount", "-o", "remount,rw", "/"])
-        subprocess.call(["sudo", "mount", "-o", "remount,rw", "/root_bypass_ramdisks/"])
         try:
+            subprocess.run(["sudo", "mount", "-o", "remount,rw", "/"], check=True)
+            subprocess.run(["sudo", "mount", "-o", "remount,rw", "/root_bypass_ramdisks/"], check=True)
             yield
         finally:
-            subprocess.call(["sudo", "mount", "-o", "remount,ro", "/"])
-            subprocess.call(["sudo", "mount", "-o", "remount,ro", "/root_bypass_ramdisks/"])
-            subprocess.call(["sudo", "mount", "-o", "remount,rw", "/root_bypass_ramdisks/etc/cups"])
+            try:
+                subprocess.run(
+                    ["sudo", "mount", "-o", "remount,ro", "/"],
+                    check=False,
+                    stderr=subprocess.DEVNULL,
+                    stdout=subprocess.DEVNULL,
+                )
+                subprocess.run(
+                    ["sudo", "mount", "-o", "remount,ro", "/root_bypass_ramdisks/"],
+                    check=False,
+                    stderr=subprocess.DEVNULL,
+                    stdout=subprocess.DEVNULL,
+                )
+                subprocess.run(
+                    ["sudo", "mount", "-o", "remount,rw", "/root_bypass_ramdisks/etc/cups"],
+                    check=False,
+                    stderr=subprocess.DEVNULL,
+                    stdout=subprocess.DEVNULL,
+                )
```

Comment:
just an idea, but would it be possible to read a value from somewhere accessible outside the code (IoT odoo configuration file ? os.environ ? ) so that if a flag is set, we don't re-remount in readonly ? This would be more convenient to develop than having to commenting this lines every time

Reply:
*Good Enough Solution*: `write_mode` alias allows you to mount read/write filesystem easily.  
(Your solution would be a good idea though).